instrumentation/go.mongodb.org/mongo-driver/v2/mongo/otelmongo/mongo_test.go

```diff
@@ -24,6 +24,75 @@ import (
 
 type validator func(sdktrace.ReadOnlySpan) bool
 
+func TestWithSpanNameFormatter(t *testing.T) {
+	tt := []struct {
+		title             string
+		operation         func(context.Context, *mongo.Database) (any, error)
+		expectedSpanName  string
+		mockResponses     []bson.D
+		SpanNameFormatter SpanNameFormatterFunc
+	}{
+		{
+			title: "insert",
```

Comment:
[nit] Could we title this w/ the context of using the default formatter?

```suggestion
			title: "insert using default SpanNameFormatter",
```